src/assets/extra.json

```diff
@@ -0,0 +1,72 @@
+[
```

Comment:
I think we want the manifest to still be pulled from release, but have a known difference in the filename, e.g. agnos.json agnos-extras.json